Ball of Duty Server/Domain/Entities/Character.cs

```diff
@@ -79,5 +97,7 @@ public override bool Destroy(GameObject exterminator)
             }
             return false;
         }
+
+        public abstract void UpdateServerStats();
```

Comment:
Well, just wanted to point out it's for updating the stats the server is responsible for, it doesn't call to the clients for them to update stats